src/test/java/com/adobe/epubcheck/cli/CheckerIT.java

```diff
@@ -1,59 +0,0 @@
-package com.adobe.epubcheck.cli;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-
-import java.io.IOException;
-import java.io.InputStream;
-
-import org.junit.Test;
-
-import com.google.common.collect.ObjectArrays;
-
-public class CheckerIT
-{
-
-  private static final String[] cmd = new String[] { "java", "-jar", "target/epubcheck.jar" };
```

Comment:
Hi @rdeltour 

Yes, I believe that this test is not moved. It's been a while since I did this pull request so I might remember the reason wrongly. 

But my best guess is that we have multiple tests checking that different command line variants works and returns the expected output so having one extra without any parameters would just fail when all the others fail. So it would not give us any extra information IMHO.

If this is an incorrect assumption then we just need to add another command line test using the function that doesn't have any parameters.

Best regards
Daniel

Reply:
Putting this as another test in the `CommandLineTest` class will not work: this test is specifically an **integration** test. It launches Java as an external process, so that we can expect the result output including bootstrap messages etc.

For instance, this was needed to test a regression with some messages produced by Saxon only at the JVM launch time. If the test is run on an existing JVM, where the EPUBCheck classes were already initialized, then these messages would no longer happen.

Reply:
For reference, this test was included in #910.

Reply:
If you see nothing wrong, I'll re-introduce this test in my integration branch [`pr/828/test-cleanup`](https://github.com/w3c/epubcheck/tree/pr/828/test-cleanup), and will then merge the PR…